pkgs/by-name/ho/homerow/package.nix

```
@@ -0,0 +1,40 @@
+{
+  lib,
+  fetchzip,
+  stdenvNoCC,
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "Homerow";
```

https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming:
> For the pname attribute:
> - It must not contain uppercase letters.

```suggestion
  pname = "homerow";
```